lib/src/pages/animations/anim6.dart

```diff
@@ -0,0 +1,165 @@
+/**
+ * Author: Ambika Dulal
+ * profile: https://github.com/Ambikadulal
+  */
+import 'package:flutter/material.dart';
+import 'package:flutter_ui_challenges/src/pages/animations/anim5.dart';
+import 'package:page_transition/page_transition.dart';
```

Comment:
Can you make it work without page_transition package (may be with existing animator package)?